BSP/Simulator/Data/killswitch-out.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Could you add multiple states to this file? Also, make sure the time is only 1 per state.